src/Helper/Utils.php

```diff
@@ -27,4 +28,26 @@ public static function fillObject($object, array $properties): void
             }
         }
     }
+
+    /**
+     * @param  array $array
+     * @param  array $requiredKeys
+     *
+     * @return array $emptyValues
+     */
+    public static function emptyValues($array, $requiredKeys = null): array
+    {
+        if (! $requiredKeys) {
```

Comment:
```php
$requiredKeys = $requiredKeys ?? array_keys($array);
```

Overigens wordt die branch niet getest in jouw tests. Hoe draai je je tests? Als het goed is rolt er een html coverage report uit, die kan je openen in je browser en dan zie je direct welke regels er wel en niet gecovered zijn. Check CONTRIBUTING.md anders ff.